openassessment/templates/openassessmentblock/edit/oa_edit_prompts.html

```diff
@@ -0,0 +1,26 @@
+{% load i18n %}
+{% spaceless %}
+<div id="oa_prompts_editor_wrapper" class="oa_editor_content_wrapper">
+    <div id="openassessment_prompt_template" class="is--hidden">
+        {% include "openassessmentblock/edit/oa_edit_prompt.html" with prompt_uuid="" prompt_description="" %}
+    </div>
+
+    <div id="openassessment_prompts_instructions" class="openassessment_tab_instructions">
+        <p class="openassessment_description">
+            {% trans "Prompts. Replace the sample text with your own text. For more information, see the ORA documentation." %}
+        </p>
+    </div>
+
+    <ul id="openassessment_prompts_list">
+        {% for prompt in prompts %}
```

Comment:
I may have missed this, but is there validation to ensure the number of prompts does not change once the question is released? I think that may something we want to consider. If the question stays the same, but some students answer in three parts, and others in one or two, it should not break the ORA2 system, but it could cause a pretty poor experience.